src/gigs-board/feature/github/integration/kanban.jsx

```diff
@@ -0,0 +1,621 @@
+/* INCLUDE: "shared/lib/uuid" */
+const uuid = () =>
+  [Date.now().toString(16)]
+    .concat(
+      Array.from(
+        { length: 4 },
+        () => Math.floor(Math.random() * 0xffffffff) & 0xffffffff
+      ).map((value) => value.toString(16))
+    )
+    .join("-");
+
+const uuidIndexed = (data) => {
+  const id = uuid();
+
+  return Object.fromEntries([[id, { ...data, id }]]);
+};
+/* END_INCLUDE: "shared/lib/uuid" */
+/* INCLUDE: "shared/mocks" */
+const communities = {
+  "zero-knowledge": {
+    overviewId: 397,
+    eventsId: 401,
+
+    icon: "https://ipfs.near.social/ipfs/bafkreiajwq6ep3n7veddozji2djv5vviyisabhycbweslvpwhsoyuzcwi4",
+
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreihgxg5kwts2juldaeasveyuddkm6tcabmrat2aaq5u6uyljtyt7lu",
+
+    title: "Zero Knowledge",
+    desc: "Building a zero knowledge ecosystem on NEAR.",
+  },
+
+  protocol: {
+    overviewId: 412,
+    eventsId: 413,
+
+    icon: "https://ipfs.near.social/ipfs/bafkreidpitdafcnhkp4uyomacypdgqvxr35jtfnbxa5s6crby7qjk2nv5a",
+
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreicg4svzfz5nvllomsahndgm7u62za4sib4mmbygxzhpcl4htqwr4a",
+
+    title: "Protocol",
+    desc: "Supporting the ongoing innovation of the NEAR Protocol.",
+
+    integrations: {
+      github: {
+        kanban: {
+          boards: {
+            "18855b9c9f2-216091d-6484800b-42593f54-6102b48a": {
+              id: "18855b9c9f2-216091d-6484800b-42593f54-6102b48a",
+
+              columns: {
+                "18855f4a93e-76a9b704-14c3ebdb-1e6c0f05-22653630": {
+                  id: "18855f4a93e-76a9b704-14c3ebdb-1e6c0f05-22653630",
+                  description: "Lorem ipsum",
+                  labelSearchTerms: ["S-draft"],
+                  title: "Draft",
+                },
+              },
+
+              dataTypesIncluded: { Issue: false, PullRequest: true },
+              description: "Latest NEAR Enhancement Proposals by status",
+              repoURL: "https://github.com/near/NEPs",
+              title: "NEAR Protocol NEPs",
+            },
+          },
+        },
+      },
+    },
+  },
+
+  tooling: {
+    overviewId: 416,
+    eventsId: 417,
+
+    icon: "https://ipfs.near.social/ipfs/bafkreie2eaj5czmpfe6pe53kojzcspgozebdsonffwvbxtpuipnwahybvi",
+
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiehzr7z2fhoqqmkt3z667wubccbch6sqtsnvd6msodyzpnf72cszy",
+
+    title: "Tooling",
+    desc: "Supporting the ongoing innovation of tooling.",
+  },
+
+  "contract-standards": {
+    overviewId: 414,
+    eventsId: 415,
+
+    icon: "https://ipfs.near.social/ipfs/bafkreiepgdnu7soc6xgbyd4adicbf3eyxiiwqawn6tguaix6aklfpir634",
+
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiaowjqxds24fwcliyriintjd4ucciprii2rdxjmxgi7f5dmzuscey",
+
+    title: "Contract Standards",
+    desc: "Coordinating the contribution to the NEAR dapp standards.",
+  },
+};
+/* END_INCLUDE: "shared/mocks" */
+
+const dataTypesLocked = {
+  Issue: true,
+  PullRequest: true,
```

Comment:
Why we lock and not allow to toggle dataTypes?

Reply:
This is an MVP requirement since we have some uncertainties with data that comes for Issues API but no time to debug it yet. It'll be unlocked with / after the next milestone release, I think

cc @ori-near

Reply:
Got it!